consensus/consensus-types/src/common.rs

```diff
@@ -298,6 +348,13 @@ impl Payload {
                 }
                 Ok(())
             },
+            (true, Payload::QuorumStoreInlineHybrid(_inline_batches, proof_with_status)) => {
+                for proof in proof_with_status.proof_with_data.proofs.iter() {
+                    proof.verify(validator)?;
+                }
+                // TODO: Do we need to make any checks to verify inline_batches?
```

Comment:
Yes, we need to verify the inline batches to make sure the digest matches the txns. I think it's easiest if you actually use a `aptos_consensus::quorum_store::types::Batch` struct and call its verify function

Reply:
I couldn't import aptos_consensus crate into consensus_types directory as it is causing cyclical dependency. I'll find a way to fix it.

Reply:
Done.